news/2018-06-02-mappers-choice-awards-2017-results.md

```diff
@@ -0,0 +1,345 @@
+
+*written by [Sorcerer](https://osu.ppy.sh/u/1913190)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/despacito.jpg)](https://osu.ppy.sh/s/665562)
+
+A sip of fruit punch, a welcoming embrace of sea breeze, a wonderful panorama of the horizon, all with the lifestyle of **[Despacito ft. Daddy Yankee](https://osu.ppy.sh/s/665562)**. The difficulties by **[Ascendance](https://osu.ppy.sh/u/2931883)** and **[Spectator](https://osu.ppy.sh/u/702598)** appear slightly easier to cater to everyone. To better emphasize the harmonious rhythm, patterns of different snaps and hitsounds are applied masterfully.
+
+*written by [alienflybot](https://osu.ppy.sh/u/636114)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/jbh.jpg)](https://osu.ppy.sh/u/4879508)
+
+Despite being less active later in the year as opposed to earlier, **[JBHyperion](https://osu.ppy.sh/u/4879508)** brings home the gold as the Grand User for MCA 2017. His commitment to osu!catch and mapping as a whole cannot be explained enough, being promoted to QAT in mid-2016 and carrying through with that duty even to this day as well as continuing to be a helpful presence in the mapping scene overall with nominations. Needless to say, **[JBHyperion](https://osu.ppy.sh/u/4879508)**'s presence throughout the year has been more than enough to land him a spot at the top as this year's Grand User.
+
+*written by [Ascendance](https://osu.ppy.sh/u/2931883)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/spec.jpg)](https://osu.ppy.sh/u/702598)
+
+**[Spectator](https://osu.ppy.sh/u/702598)** is possibly a paragon of mapping to everyone in the community. Spectator never fails to impress all of us with his consistent, aesthetic mapping style, with **[Cycle Hit](https://osu.ppy.sh/s/692367)** and **[Boogie](https://osu.ppy.sh/s/569985)** being the most convincing arguments. Sitting proudly on the throne as the Elite Mapper: Aspirant, Spectator went further to produce top-notch maps with technical elements such as **[Marigold](https://osu.ppy.sh/s/605710)** and **[Stellar](https://osu.ppy.sh/s/699509)**. We look forward to seeing more eye-opening projects with superb standards from him!
+
+*written by [alienflybot](https://osu.ppy.sh/u/636114)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/nokashi.jpg)](https://osu.ppy.sh/u/5431196)
+
+An incredibly well deserved winner, **[Nokashi’s](https://osu.ppy.sh/u/5431196)** modding efforts in 2017 were absolutely top tier. With very few maps to call his own, Nokashi has selflessly put his time and effort on osu into improving and polishing other mappers creations instead. With thorough mods, great attention to detail and a very friendly attitude he even became a Beatmap Nominator in late 2017. His overall attitude and dedication is really something that the catch mapping community should be inspired by.
```

Comment:
apostrophe + s goes outside of the link. also, your call whether you want to get rid of these "smart quotes" and apostrophes